tokio-trace-core/src/metadata.rs

```diff
@@ -141,15 +140,15 @@ pub struct Meta<'a> {
 }
 
 /// Describes the level of verbosity of a `Span` or `Event`.
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
I elected to leave the `Eq` and `Ord` derives here, since having an ordering is potentially useful for filters, and there are enough variants to test that having `is_x` methods for each one seemed rather painful. WDYT?